frontend/src/scenes/insights/Insights.tsx

```diff
@@ -385,10 +392,17 @@ export function Insights(): JSX.Element {
 }
 
 function FunnelInsight(): JSX.Element {
-    const { stepsWithCount, isValidFunnel, stepsWithCountLoading } = useValues(funnelLogic({}))
+    const {
+        stepsWithCount,
+        isValidFunnel,
+        stepsWithCountLoading,
+        filters: { display },
+    } = useValues(funnelLogic({}))
+    const { featureFlags } = useValues(featureFlagLogic)
+    const shouldSetFixedHeight = !featureFlags[FEATURE_FLAGS.FUNNEL_BAR_VIZ] || display === ACTIONS_LINE_GRAPH_LINEAR
 
     return (
-        <div style={{ height: 300, position: 'relative' }}>
+        <div style={shouldSetFixedHeight ? { height: 300, position: 'relative' } : {}}>
```

Comment:
This line reads like a bit of magic. I feel there's some context missing here that I can't read from the code. Why should what be set how?

Reply:
The logic is essentially this:
- Fixed container height is always required for trendline graph
- Disable fixed height (so that container stretches vertically) for funnel steps view.

The negation of a feature flag might make it confusing. Should this be a code comment or a more major rework?

Reply:
The confusing bit is: doesn't this enable fixed height for all graphs if the flag is off? And if that's the case, why is the funnel feature flag affecting the display of actions line graphs? That seems off. I'd understand if we're somehow also checking "(!flag && display == funnel) || display == linegraph" or so, but just "!flag || display == linegraph" feels off. Is there a bug here?

Reply:
Well, the current default is to set fixed height for everything, and the funnels + feature flag is actually a special exception.  Updated the code to reflect this. (The bug you noticed here, and that is now fixed, was that it would actually _unset_ fixed height for all graphs when it should be enabled.)